### PR TITLE
Update titles and category for lines and polygon, align with gl-js implementation.

### PR DESCRIPTION
**Pull request by ank27**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [x] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### Summary of changes

- Update the title, description, packages and categories for lines and polygons examples in order to align with ios and gl-js
- Align Draw a polygon with gl-js implementation (https://docs.mapbox.com/mapbox-gl-js/example/geojson-polygon/).
lines_and_polygons
draw_polygon

### CODE REVIEW

File: app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawGeoJsonLineActivity.kt [hunk 1]
```diff
@@ -19,18 +21,26 @@ class DrawGeoJsonLineActivity : AppCompatActivity() {
   public override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_dds_draw_geojson_line)
+    mapView.getMapboxMap().setCamera(
+      CameraOptions.Builder().center(
+        Point.fromLngLat(
+          LATITUDE,
+          LONGITUDE
+        )
+      ).zoom(CAMERA_ZOOM).build()
```
Comment (kiryldz) [inline app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawGeoJsonLineActivity.kt:30]: Perhaps just `ZOOM` to align with `LATITUDE` and `LONGITUDE`?

File: app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawPolygonActivity.kt [hunk 2]
```diff
@@ -25,17 +26,26 @@ class DrawPolygonActivity : AppCompatActivity() {
...
     setContentView(R.layout.activity_dds_draw_polygon)
+    mapView.getMapboxMap().setCamera(
+      START_CAMERA_POSITION
+    )
     mapView.getMapboxMap().loadStyle(
-      style(styleUri = Style.MAPBOX_STREETS) {
+      style(styleUri = Style.LIGHT) {
         +geoJsonSource(SOURCE_ID) {
-          feature(Feature.fromGeometry(Polygon.fromLngLats(POINTS)))
+          url("asset://maine_polygon.geojson")
```
Comment (kiryldz) [inline app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawPolygonActivity.kt:35]: I'd put url to const

File: app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawGeoJsonLineActivity.kt (hunk 1 above)
Comment (kiryldz) [reply app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawGeoJsonLineActivity.kt:30]: or even use approach of setting camera as a whole obj from `DrawPolygonActivity.kt`

**Review by kiryldz — APPROVED** [review]
LGTM with couple nits

**Review by Chaoba — APPROVED** [review]
LGTM

File: app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawGeoJsonLineActivity.kt (hunk 1 above)
Comment (ank27) [reply app/src/main/java/com/mapbox/maps/testapp/examples/linesandpolygons/DrawGeoJsonLineActivity.kt:30]: Yeah, here I just wanted to see what are different ways to get camera options.

**Pull request merged**